### ISSUE TITLE
Product page on BO - missing area with Save button and other missing or not working areas

### ISSUE THREAD

**Issue by iliankirov** [PrestaShop/PrestaShop#22844]
#### Describe the bug

When try to update product information on BO > Catalog > Product:
- the images area doesn't work, and it don't look as usual (there is a text above the images previews), notting happened when click in add image button, can't rearange images, can't remove images, can't change cover image.
- the "Combinations" tab is missing in tab area;
- the "Delete" buttons in Features ares doesn't works.
- the area with buttons: "Preview", "Online", "Save", "Duplicate" and etc. is missing.
The problem doesn't depends from the browser, I'm using Chrome, tested on Mozila, Edge, IE. I noticed, that when the page loads correctly, in the html there is java script code, which one in missing, when there are missing or don't working elements.

#### Expected behavior

- the images area to allow me to upload new image by clicking on "+" square, to can rearange images and change cover image;
- the "Combination" tabl to be next to the "Basic settings" tab;
- to deleve the Features, when click on their "Delete" buttons;
- the area with buttons: "Preview", "Online", "Save", "Duplicate" and etc. to be visible.

#### Steps to Reproduce

1. Go to 'BO'
2. Click on 'Catalon > Product'
4. See error, that I described. Sometimes it happened the page to be visualised wright, or after clear the cashe and/or refresh some time the page become working. But it's possible after clearing the cashe and reloading the page many times, it contiue to be not working properly.

**Screenshots**

Screenshot, when doesn't work properly:

![scr1](https://user-images.githubusercontent.com/65872504/104663460-b8992880-56d5-11eb-9e68-a2a311096394.jpg)

Screenshot, when working properly with marked problem areas/elements:

![scr2](https://user-images.githubusercontent.com/65872504/104662148-d5802c80-56d2-11eb-87e7-f691df228685.jpg)

Screen with error, when Debug mode is activated:

![New Picture](https://user-images.githubusercontent.com/65872504/104662577-d4033400-56d3-11eb-820c-35025ad2ca0b.jpg)

Screenshop with store info (PHP version, MariaDB version and etc.):

![New-Picture-(3)](https://user-images.githubusercontent.com/65872504/104662301-33147900-56d3-11eb-921d-223002ce8393.jpg)

Here is error log file:

[error_log.txt](https://github.com/PrestaShop/PrestaShop/files/5817578/error_log.txt)

#### Additional information

* PrestaShop version: 1.7.6.5
* PHP version: 7.2.34

Thank you!

**Comment by yannicka** [comment]
This is a common problem: you only allow 128 MB of RAM (in "Memory limit").

PrestaShop recommends a minimum of 256 MB: <https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/>.

**Comment by florine2623** [comment]
Hi @iliankirov ,

Please make sure your server config meets all PrestaShop's requirement by following @yannicka 's link.
Thanks!

**Comment by iliankirov** [comment]
Thank you for all your advices! I tryed the things that you adviced me. I tested with the changes of memory_limit directive, setting it on : 256M, 512M, 1024M, but there was no changes. Now it's set on 512M, the cashe is cleared a lot of times, but the page works in the same way. 

Here is all the directives, that I changed:
memory_limit = 512M      ;
max_execution_time = 700     ;
max_input_time = 420	;
allow_url_fopen = On     ;
magic_quotes_gpc = Off     ;

![New Picture (5)](https://user-images.githubusercontent.com/65872504/104784438-ef804480-5790-11eb-96c8-1e6fb6bafe0a.jpg)


![New Picture (4)](https://user-images.githubusercontent.com/65872504/104784443-f3ac6200-5790-11eb-9e74-558409e7badd.jpg)
